components/director/internal/open_resource_discovery/validation.go

```diff
@@ -681,14 +680,15 @@ func validateEventResourceDefinition(value interface{}, event model.EventDefinit
 		return nil
 	}
 
-	eventResourceDef, ok := value.([]*model.EventResourceDefinition)
+	_, ok := value.([]*model.EventResourceDefinition)
 	if !ok {
 		return errors.New("error while casting to EventResourceDefinition")
 	}
 
-	if len(eventResourceDef) == 0 {
-		return errors.New("when event resource visibility is public or internal, resource definitions must be provided")
-	}
+	// soften validation temporarily
+	// if len(eventResourceDef) == 0 {
+	//	return errors.New("when event resource visibility is public or internal, resource definitions must be provided")
```

Comment:
I think the better approach for softening is not to remove the code snippet (comment out) but to log the error instead returning it here. 
With this the errors will still be visible in logs and we will know when to harden the rule again (when no more errors are logged)

Reply:
Added logging of the error with default logger.